import_data/invoke.yaml

```diff
@@ -26,6 +26,21 @@ osm:
   ## Path of the file
   file:
 
+wikidata:
+  sitelinks:
+    url: https://github.com/QwantResearch/wikimedia-dumps/releases/download/10/2019/sitelinks.csv.gz
+    file: sitelinks.csv.gz
+    table: wd_sitelinks
+
+  labels:
+    url: https://github.com/QwantResearch/wikimedia-dumps/releases/download/10/2019/labels.csv.gz
+    file: labels.csv.gz
+    table: wd_names
+
+wikimedia_stats:
+  url: https://github.com/QwantResearch/wikimedia-dumps/releases/download/10/2019/stats.csv.gz
+  file: stats.csv.gz
+  table: wm_stats
```

Comment:
I think we will also need an option to enable/disable these imports, at least for debug purposes.

Reply:
I put it as disabled by default, maybe I shoulded ?

The import isn't that long, it takes about 1 hour to import labels (the operation on hstores takes a bit more time) and the two other tasks only need a few minutes to run.